plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -147,7 +241,8 @@ public void testTrinoCreatedParameterizedVarchar()
                 .addRoundTrip("varchar", "'unbounded'", createUnboundedVarcharType(), "CAST('unbounded' AS varchar)")
                 .addRoundTrip("varchar(10)", "NULL", createVarcharType(255), "CAST(NULL AS varchar(255))")
                 .addRoundTrip("varchar", "NULL", createUnboundedVarcharType(), "CAST(NULL AS varchar)")
-                .execute(getQueryRunner(), trinoCreateAsSelect("trino__test_parameterized_varchar"));
+                .execute(getQueryRunner(), trinoCreateAsSelect("trino__test_parameterized_varchar"))
+                .execute(getQueryRunner(), trinoCreateAndInsert("trino__test_parameterized_varchar"));
```

Comment:
Let's rename the table to `trino_test_parameterized_varchar`. (remove duplicated hyphen)

Reply:
done